lib/tasks/backfill_fraud_pending_reason.rake

```diff
@@ -1,28 +1,65 @@
 namespace :profiles do
   desc 'If a profile is in review or rejected, store the reason it was marked for fraud'
 
+  ##
+  # Usage:
+  #
+  # Print pending updates
+  # bundle exec rake profiles:backfill_fraud_pending_reason
+  #
+  # Commit updates
+  # bundle exec rake profiles:backfill_fraud_pending_reason UPDATE_PROFILES=true
+  #
   task backfill_fraud_pending_reason: :environment do |_task, _args|
     ActiveRecord::Base.connection.execute('SET statement_timeout = 60000')
 
+    update_profiles = ENV['UPDATE_PROFILES'] == 'true'
+
     profiles = Profile.where(
       fraud_pending_reason: nil,
     ).where(
       'fraud_review_pending_at IS NOT NULL OR fraud_rejection_at IS NOT NULL',
     )
+
     profiles.each do |profile|
-      fraud_pending_reason = case profile.proofing_components&.[]('threatmetrix_review_status')
+      proofing_component_status = profile.proofing_components&.[]('threatmetrix_review_status')
+      fraud_pending_reason = case proofing_component_status
                              when 'review'
                                'threatmetrix_review'
                              when 'reject'
                                'threatmetrix_reject'
                              else
                                'threatmetrix_review'
```

Comment:
Should we warn here?

Reply:
This is part of the reason I added the dry run feature. It will print the raw result so we can do a check ourselves for values that aren't review or reject. I don't expect any.